.pre-commit-config.yaml

```diff
@@ -82,3 +82,10 @@ repos:
         args: [--fix, --show-fixes]
     # Run the formatter.
     -   id: ruff-format
+
+
+-   repo: https://github.com/PyCQA/doc8
+    rev: v1.0.0
+    hooks:
+    -   id: doc8
+        args: [--config=pyproject.toml, nilearn]
```

Comment:
So IIUC, would this fix the line violations when committing?

Reply:
no doc8 has no auto fix option: you would have to manually fix it

Reply:
I see so it would just throw an error while committing